RELEASE.md

```diff
@@ -1,6 +1,12 @@
 # Release History
 *****************
 
+## Release ONDEWO NLU APIS 1.1.1
```

Comment:
Version *.*.x are patch versions.
These should only contain minimal changes that are unrelated to the interface usage.

Things like, improving comments, README, automatic setups, testing and so on.
Every change that changes the interface (or adds on top) is at least a `minor` change, affecting the second number in the versioning.

Therefore, we should be writing the Release notes of 1.2.0 instead :)

Reply:
Ok, got it!